tests/test_no_hardcoded_cjk.py

```
@@ -92,6 +92,24 @@ def _is_allowed(rel: str) -> bool:
 
 
 def _iter_source_files():
+    # Scan only git-TRACKED files: the rule governs the committed codebase,
+    # not local untracked/vendored experiments (which also never reach CI).
+    import subprocess
+    try:
+        out = subprocess.run(
+            ["git", "ls-files", "-z"],
```

Ruff (0.15.14)

[error] Starting a process with a partial executable path

(S607)